src/applications/lgy/coe/shared/components/ReviewAndDownload.jsx

```diff
@@ -1,31 +1,28 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+
+import environment from 'platform/utilities/environment';
 
 import DownloadLink from './DownloadLink';
 
 const downloadLinkLabel = 'Download your COE (PDF) 0.20MB';
 
-const ReviewAndDownload = ({ downloadUrl }) => (
+const downloadUrl = `${environment.API_URL}/v0/coe/download_coe`;
```

Comment:
We are able to remove the downloadUrl from the Redux store and the other components because we can now call `vets-api` directly from here